tests/Platforms/SQLitePlatformTest.php

```diff
@@ -173,20 +174,20 @@ public function testGeneratesTypeDeclarationForBigIntegers(): void
 
     public function getGenerateIndexSql(): string
     {
-        return 'CREATE INDEX my_idx ON mytable (user_name, last_login)';
+        return 'CREATE INDEX "my_idx" ON mytable ("user_name", "last_login")';
     }
 
     public function getGenerateUniqueIndexSql(): string
     {
-        return 'CREATE UNIQUE INDEX index_name ON test (test, test2)';
+        return 'CREATE UNIQUE INDEX "index_name" ON test ("test", "test2")';
     }
 
     public function testGeneratesIndexCreationSqlWithSchema(): void
     {
         $indexDef = new Index('i', ['a', 'b']);
 
         self::assertSame(
-            'CREATE INDEX main.i ON mytable (a, b)',
+            'CREATE INDEX main."i" ON mytable ("a", "b")',
             $this->platform->getCreateIndexSQL($indexDef, 'main.mytable'),
```

Comment:
The index name is quoted because it comes from `Index`, and we know that it's an unqualified name. The table name comes from `'main.table'`, which is SQL. It's unquoted there, so it's unquoted in the resulting SQL as well. This isn't correct but is out of scope.